add-features-with-contingent-values/src/main/res/layout/add_feature_layout.xml

```diff
@@ -26,34 +25,33 @@
                     android:layout_height="wrap_content"
                     android:fontFamily="sans-serif-thin"
                     android:text="Add Feature"
-                    android:textColor="@android:color/white"
                     android:textSize="17sp"
                     android:textStyle="bold"
                     app:layout_constraintBottom_toBottomOf="parent"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
                     app:layout_constraintTop_toTopOf="parent" />
 
-                <android.widget.TextView
+                <com.google.android.material.button.MaterialButton
                     android:id="@+id/applyTv"
+                    style="?attr/materialButtonOutlinedStyle"
```

Comment:
Use custom style, instead of default style. (line 37 & 49)

```suggestion
                    style="@style/OutlineButton"
```

Reply:
I think we should use the default style here, as it will ultimately use the custom style per the samples lib styles.xml
Else we might end up updating at all places in the samples(for consistency), which use the default style to use the custom style in their layout. 
and the style file change ` <item name="materialButtonOutlinedStyle">@style/OutlineButton</item>` would be redundant?